CHANGELOG.md

```diff
@@ -2,6 +2,236 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [0.0.2](https://github.com/JamesLMilner/terra-draw/compare/v0.0.1...v0.0.2) (2023-10-01)
```

Comment:
We don't need to do a release or update the CHANGELOG for this change. I'll show you how we can undo this!